app/javascript/components/Greeting.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { connect } from 'react-redux';
+import { createStructuredSelector } from "reselect";
+
+const GET_MESSAGES_REQUEST = 'GET_MESSAGES_REQUEST';
+const GET_MESSAGES_SUCCESS = 'GET_MESSAGES_SUCCESS';
+
+function getMessages() {
+  console.log("Action!");
+  return async dispatch => {
+    dispatch({ type: GET_MESSAGES_REQUEST });
+    try {
+      const response = await fetch(`http://localhost:3000/v1/message`);
+      const json = await response.json();
+      console.log("JSON: " + json);
+      return dispatch(getMessagesSuccess(json));
+    } catch (error) {
+      return console.log(error);
+    }    
+  };
+}
+
+export function getMessagesSuccess(json) {
+  return {
+    type: GET_MESSAGES_SUCCESS,
+    json
+  }
+}
+
+class Greeting extends React.Component{
+  render() {
+
+    const { messages } = this.props;
+    console.log(messages);
```

Comment:
- [ ] <b>REQUIRED:</b> Kindly keep your codebase clean by making sure to remove all debugging code like `console.log` when you no longer need them.